packages/snaps-controllers/src/snaps/registry/registry.ts

```diff
@@ -27,6 +27,8 @@ export type SnapsRegistry = {
     snaps: SnapsRegistryRequest,
   ): Promise<Record<ValidatedSnapId, SnapsRegistryResult>>;
 
+  updateDatabase(): Promise<void>;
```

Comment:
These are public methods. The newly introduced method is not public.

Reply:
There are not public methods, all of them are prefixed with a `#` in the controller